app/services/builders/review_turnaround.rb

```diff
@@ -11,7 +13,17 @@ def call
     private
 
     def calculate_turnaround
-      @review_request.reviews.first.opened_at.to_i - @review_request.created_at.to_i
+      (opened_review_request.to_i - @review_request.created_at.to_i) - weekend_days_as_seconds
+    end
+
+    def weekend_days_as_seconds
+      range_days = @review_request.created_at.to_date..@review_request.reviews.first.opened_at.to_date
+      weekends = range_days.select { |day| day.wday == 6 || day.wday ==0 }.count
+      weekends * SECONDS_IN_A_DAY
```

Comment:
not sure if we care about it but if someone submits a review on a saturday or sunday this might not work fine